### PR TITLE
refactor: replace Nusi FormBuilder

### PR DESCRIPTION
**Pull request by hujiahao-hjh**
## What this PR does / why we need it:
Replace Nusi FormBuilder.

## Does this PR introduce a user interface change?

❎ No

## ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |              |
| 🇨🇳 中文    |              |

## Does this PR need be patched to older version?
❎ No

## Which issue(s) this PR fixes:

Fixes #

### CODE REVIEW

File: core/src/common/components/form-builder/form-builder.tsx [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+ * isMultiColumn: whether to use multiple column or not
+ * columnNum: amount of column, only become effective when isMultiColumn is true
+ * readonly: whether all the Form.Item in Form is readonly, default false.
+ * else: the same as antd Form
+ *
+ * */
+export interface IContextType {
+  realColumnNum?: number;
+  parColumnNum?: number;
+  parIsMultiColumn?: boolean;
```
Comment (McDaddy) [inline core/src/common/components/form-builder/form-builder.tsx:34]: what does `par` mean?

File: core/src/common/components/form-builder/form-builder.tsx [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+/* eslint-disable no-param-reassign */
+import React from 'react';
+import { Form } from 'nusi';
+import { FormInstance, FormProps } from 'antd/lib/form/Form';
+import ResizeObserver from 'rc-resize-observer';
+import { throttle } from 'lodash';
+import { IFieldType } from './fields';
+
+/**
+ * FormBuilder is a Form container.
```
Comment (McDaddy) [inline core/src/common/components/form-builder/form-builder.tsx:23]: move this annotation to `IProps`

File: core/src/common/components/form-builder/form-builder.tsx (hunk 1 above)
Comment (hujiahao-hjh) [reply core/src/common/components/form-builder/form-builder.tsx]: it means parent.

File: core/src/common/components/form-builder/fields.tsx [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+                className,
+                wrapperClassName,
+                label,
+                isHoldLabel = true,
+                colSpan,
+                ...rest
+              } = item;
+              const afterAddRequiredRules =
+                required && !some(rules, (rule) => has(rule, 'required'))
+                  ? [{ required: true, message: `${label}${i18n.t('can not be empty')}` }, ...rules]
```
Comment (laojun) [inline core/src/common/components/form-builder/fields.tsx:115]: `${label}${i18n.t('can not be empty')}`  don't concatenate words with i18n

File: core/src/common/components/form-builder/fields.tsx [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+        const fieldRealColumnNum = getColumn(contextProps);
+        if (!fieldRealColumnNum) return null;
+        return (
+          <Row gutter={[20, 0]}>
+            {map(fields, (item, idx) => {
+              const {
+                type: Comp,
+                customProps = {},
+                required = true,
+                rules = [],
```
Comment (Rayzhangzhang) [inline core/src/common/components/form-builder/fields.tsx:104]: `validator` has been replaced with `rules` in FormBuilder. Check if there still exits `validator` in fields.

**Review by McDaddy — APPROVED** [review]

**Pull request merged**